### ISSUE TITLE
Keyboard shortcut for sending file to Octoprint

### ISSUE THREAD

**Issue by soerup** [prusa3d/PrusaControl#124]
There is not keyboard short cut for sending sliced file - so you still need to grab the mouse after "Ctrl-R" for Slice

**Comment by bubnikv** [comment]
Fair enough. Would you please propose one? Please make sure that it is not
used by another function. Ctrl-H sounds nice, but I am a bit worried that
it may be confused with Help.

**Comment by larsarea9** [comment]
CTRL-T (doesn't make mnemoic easy, but close to CTRL-R for easy forwarding
